### ISSUE TITLE
Several versions of JSON-LD context?

### ISSUE THREAD

**Issue by aisaac** [IIIF/website#273]
While checking the implementation of the new [mapping for "profile" and related elements](https://github.com/IIIF/api/issues/1702) I found that the official JSON context is at
https://iiif.io/api/presentation/3/context.json
but there is still an old context at
https://iiif.io/api/presentation/3/combined-context.json
which actually doesn't reflect PR IIIF/api#1722 (considering that this PR seems to be for the 'combined context' file).

**Comment by azaroth42** [comment]
Thanks Antoine!

This seems to be a deployment issue, as the combined-context file is not in the master branch:
   https://github.com/IIIF/api/tree/master/source/api/presentation/3

Transferred the issue to website and assigned to Glen for investigation.

**Comment by glenrobson** [comment]
Hi Rob, is this just a case of deleting the `combined-context.json`? (just checking I've understood correctly).

**Comment by azaroth42** [comment]
Yes ... but that's just a symptom of the issue. The file isn't in the master branch any more and hence shouldn't be being deployed. Does the deployment script only update files, and not check for deletions?

**Comment by glenrobson** [comment]
Your right the deployment script only adds and updates not delete. I've manually deleted it for now but will look into fixing the deployment scripts.
